packages/edit-site/src/components/global-styles/shadows-edit-panel.js

```diff
@@ -223,6 +223,7 @@ export default function ShadowsEditPanel() {
 						} }
 					>
 						<InputControl
+							__next40pxDefaultSize
```

Comment:
Global Styles ▸ Custom ▸ Add new ▸ "Rename" action in three-dot menu.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/d464e370-e620-4c4a-86f8-7bb2ee4ddbd7" alt="Rename shadow modal, after" width="417">|<img src="https://github.com/user-attachments/assets/a557bab1-d5c5-4bd7-8672-775b8fce52f6" alt="Rename shadow modal, before" width="417">|